dll/win32/crtdll/crtdll.spec

```diff
@@ -1,528 +1,528 @@
-# Old C runtime library. All functions provided by msvcrt
+# Old C runtime library. All functions provided by
```

Comment:
provided by?

This effectively changes crtdll from a forwarded to a full-blown (more or less) copy of msvcrt?

Reply:
Provided by itself, will fix.

Yes, more or less. But this is important : a program compiled with crtdll (if you find any) is not supposed to mess around with msvcrt state, and our libraries should not be at the mercy of a program using crtdll messing around with it.

Not that we should rely much on MSVCRT state neither, but this mitigates one risk :-/

Reply:
Please, put a reference to a previous commit in a commit message f0c91ae971ca9affd819aa10cf65ad925217932b

Reply:
allright, thanks for the explanation!

Reply:
> Please, put a reference to a previous commit in a commit message [f0c91ae](https://github.com/reactos/reactos/commit/f0c91ae971ca9affd819aa10cf65ad925217932b)

Done